README.md

```diff
@@ -5,6 +5,7 @@
 [![CircleCI](https://circleci.com/gh/intuit/user-data-for-fraud-prevention/tree/master.svg?style=shield)](https://circleci.com/gh/intuit/user-data-for-fraud-prevention/tree/master)
 ![Contributions welcome](https://img.shields.io/badge/contributions-welcome-orange)
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+![code coverage: statements](coverage/badge-statements.svg)
```

Comment:
Badge does not seem to be available. I think the right link from circle ci has to be used.
<img width="667" alt="Screen Shot 2020-10-01 at 22 27 33" src="https://user-images.githubusercontent.com/4538858/94865398-689f9a00-0435-11eb-88cc-bc98473a14d5.png">

Reply:
Also all the four code coverage badges have to be added. Like this:
<img width="660" alt="Screen Shot 2020-09-30 at 22 37 25" src="https://user-images.githubusercontent.com/4538858/94865546-b0262600-0435-11eb-971f-f9e6d2a03f4a.png">